### ISSUE TITLE
Find out if there is a way to remove absolute paths from the xcode project

### ISSUE THREAD

**Issue by jeantil** [jeantil/autokbisw#3]

**Comment by bao3** [comment]
I know th Path issue, look the picture 

![图片](https://i.loli.net/2017/12/20/5a3a096ea57dc.jpg)

you can change it to `/usr/local/bin`，at the same time you should change the destination to `Absolute Path`

![图片](https://i.loli.net/2017/12/20/5a3a0a5f64d10.jpg)

And it worked .
But I suggest to build it to `Release` not ` Debug` .

**Comment by bao3** [comment]
Resolved, change your file in `./autokbisw.xcodeproj/project.pbxproj` 

```
/* Begin PBXCopyFilesBuildPhase section */
		0F81A5021DE48BA6007372C9 /* CopyFiles */ = {
			isa = PBXCopyFilesBuildPhase;
			buildActionMask = 12;
			dstPath = /usr/local/bin;
			dstSubfolderSpec = 0;
			files = (
				0F2C43CC1DE4B6A20076D74F /* autokbisw in CopyFiles */,
			);

```

This will make install automatically to `/usr/local/bin` after you build it.

**Comment by jeantil** [comment]
Hi bao3, 

Thanks for having a look into this ! However you solution is also using an absolute path which I am trying to avoid. I have a strong preference for selfcontained builds which do not accidentaly write outside their root directory.

>This will make install automatically to /usr/local/bin after you build it.

this is not acceptable to me :) I don't want the binary to be written to a system global location after each build. This should only happen when a user explicitely requests an installation. 
I will have some free time to go through these issues and find a satisfying solution in January :)

**Comment by jeantil** [comment]
fixed in #20
